app/components/sidebars/main/channels_list/list/list.js

```diff
@@ -414,16 +400,24 @@ export default class List extends PureComponent {
     };
 
     render() {
+        const collapsedThreadsEnabled = true;
         const {testID, styles, theme} = this.props;
         const {sections, showIndicator} = this.state;
 
         const paddingBottom = this.listContentPadding();
+        const indicatorStyle = [styles.above];
+        if (collapsedThreadsEnabled) {
+            indicatorStyle.push({marginTop: 70});
+        }
 
         return (
             <View
                 style={styles.container}
                 onLayout={this.onLayout}
             >
+                {collapsedThreadsEnabled && (
+                    <ThreadsSidebarEntry/>
+                )}
                 <SectionList
```

Comment:
Tried using `stickyHeaderIndices` but it didn't work so ended up placing the component above the `SectionList`